codespell_lib/data/dictionary.txt

```diff
@@ -5398,8 +5398,11 @@ charactrs->characters
 characts->characters
 characture->character
 charakter->character
+charakters->characters
 chararacter->character
+chararacters->characters
 chararcter->character
+chararcters->characters
 charas->chars
 charascters->characters
```

Comment:
Do you want to add the singular here?

And there are some plurals missing for these two too:
charactet->character
characteter->character

Reply:
I only inserted the first one, because the latest two entries in the plural form are already present in the dictionary:
https://github.com/codespell-project/codespell/blob/32857cc6a9bee6072706133c42068d09491839c3/codespell_lib/data/dictionary.txt#L5390
https://github.com/codespell-project/codespell/blob/32857cc6a9bee6072706133c42068d09491839c3/codespell_lib/data/dictionary.txt#L5385

Reply:
Oops thanks!